docs/php/how-to-set-a-workflow-run-timeout-in-php.md

```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-set-a-workflow-run-timeout-in-php
+title: How to set Workflow Run Timeout in PHP
+sidebar_label: Workflow Run Timeout
+---
+
+`workflowRuntTimeout` runs timeout limits duration of a single Workflow invocation.
```

Comment:
Check the init caps on these; in the example code it shows `WorkflowRunTimeout`.